hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -480,6 +480,20 @@ public class HoodieWriteConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation(BulkInsertSortMode.class);
 
+  public static final ConfigProperty<Boolean> INSERT_SORT = ConfigProperty
```

Comment:
Done. IIUC, you are asking to use `BulkInsertSortMode::NONE` and `BulkInsertSortMode::GLOBAL_SORT` (instead of a boolean). FYI,  there are no sort modes for `insert`. There is only global sort (i.e sort the entire input batch). Hence the valid values are NONE or GLOBAL_SORT

Reply:
lets make sure we throw an exception for the unsupported mode.

Reply:
Already handled by setting valid values for the config property.